lbuild/module.py

```diff
@@ -396,6 +397,12 @@ def __init__(self,
         self.options = {}
 
     @property
+    def _logger(self):
+        if self.__logger is None and self._fullname is not None:
+            self.__logger = logging.getLogger(self._fullname)
```

Comment:
Can you add a prefix to make it easier to distinguish between lbuild logging entries and user script logging entries? Otherwise it might get confusing.